src/cascadia/CascadiaPackage/Package-Dev.appxmanifest

```diff
@@ -99,7 +99,7 @@
             <com:ComInterface>
                 <com:ProxyStub Id="DEC4804D-56D1-4F73-9FBE-6828E7C85C56" DisplayName="OpenConsoleHandoffProxy" Path="OpenConsoleProxy.dll"/>
                 <com:Interface Id="E686C757-9A35-4A1C-B3CE-0BCC8B5C69F4" ProxyStubClsid="DEC4804D-56D1-4F73-9FBE-6828E7C85C56"/>
-                <com:Interface Id="59D55CCE-FC8A-48B4-ACE8-0A9286C6557F" ProxyStubClsid="DEC4804D-56D1-4F73-9FBE-6828E7C85C56"/>
+                <com:Interface Id="AA6B364F-4A50-4176-9002-0AE755E7B5EF" ProxyStubClsid="DEC4804D-56D1-4F73-9FBE-6828E7C85C56"/>

```

Comment:
OK SO, we need to take very special care with the interfaces here. Since there is only ONE PROXY STUB no matter how many instances of Terminal you have installed, _we'll need to keep all the old interfaces in it for the versions of Terminal in active use._ This is so that simply installing Preview doesn't blow away the ability for Stable to QI for  IIDs that existed when Stable was new. Yes. I know.

Reply:
yikes
YIKES

Reply:
Oh no, so will we need to rev this to `ITerminalHandoffN+1` every time we want to add args to `TERMINAL_STARTUP_INFO`? If that's the case then I'll need to put the full set of args in here now, even if I'm not collecting them yet.

Reply:
Honestly, I'm not that broken up about having to rev the interface (you'll also have to rev the struct and give it a version, so you may want to get out ahead of that)

Reply:
~Is it madness to start with `TERMINAL_STARTUP_INFO_2`, so that the struct always matches~

yes, yes it is, we might not always rev the struct when we rev the interface